fedora-coreos/provisioning/main.tf

```diff
@@ -34,15 +36,15 @@ resource "aws_instance" "fcos-lab-instance" {
   tags = {
     Name = "fcos-lab"
   }
-  ami           = "ami-0ea3c2efdcead938c"
+  ami           = "ami-06128ecf4b4101217"
```

Comment:
maybe port [this feature](https://github.com/coreos/coreos-tutorials/commit/97376716dce47080dd9e8f016753c55c5c79da96) here ?

Reply:
Completed with https://github.com/coreos/coreos-tutorials/commit/d9ea4e5f3d1cd18f80e1c3334ae7aa4f492af392.